Controllers/MoviesController.cs

```diff
@@ -35,10 +41,35 @@ public MoviesController(DbContextOptions<DataBaseContext> options)
             }
             movieGenreVM.genres = new List<SelectListItem>();
 
+            switch (sortOrder)
```

Comment:
with this switch, the method is so large, try to put this switch in another methdo, and call it from here